file_parse.go

```diff
@@ -0,0 +1,57 @@
+package samosa
+
+import (
+	"bufio"
+	"os"
+	"strconv"
+	"strings"
+)
+
+// read the file in lines and load in to map
+func readCoverReport(f string, covReport map[string]int) error {
+	file, err := os.Open(f)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
```

Comment:
@ehrktia I'm confused on why do we need this file? I'm not seeing any references to these helper functions, except tests. Would you like me to proceed with removing this?

Reply:
@burntcarrot sorry we do not need this one , you can clean it up .